pkg/apis/core/v1alpha1/types_overridepolicy.go

```diff
@@ -65,17 +65,17 @@ type OverridePolicyStatus struct {
 }
 
 type TargetClusters struct {
-	// ClusterNames selects clusters by their names.
-	// Empty ClusterNames selects all clusters.
+	// Clusters selects FederatedClusters by their names.
+	// Empty Clusters selects all FederatedClusters.
 	// +optional
-	ClusterNames []string `json:"clusterNames,omitempty"`
+	Clusters []string `json:"clusters,omitempty"`
 
-	// ClusterSelector selects clusters by their labels.
-	// Empty labels selects all clusters.
+	// ClusterSelector selects FederatedClusters by their labels.
```

Comment:
Is there a reason for changing all `cluster`s to `FederatedCluster`s? I feel in the context of kubeadmiral `cluster` should unambiguously refer to `FederatedCluster`s.

Reply:
> Is there a reason for changing all `cluster`s to `FederatedCluster`s? I feel in the context of kubeadmiral `cluster` should unambiguously refer to `FederatedCluster`s.

The change here is because after changing the `clusterNames` to `clusters`, the relevant comments will become `Clusters selects Clusters by their names.`, which seems a bit ambiguous, and the change to `FederatedClusters` is mainly to clarify that the first `clusters` in the comment is used as a field Name, and `FederatedClusters` as the selected K8S object.

Reply:
lgtm, it's just a comment change anyway